.github/workflows/samples_getstarted_flowasfunction.yml

```diff
@@ -0,0 +1,52 @@
+# This code is autogenerated.
+# Code is generated by running custom script: python3 readme.py
+# Any manual changes to this file may cause incorrect behavior.
```

Comment:
please make this ci pass

Reply:
The workflow passed, but will disable since compliance check does not allow --extra-index-url in repo

Reply:
![image](https://github.com/microsoft/promptflow/assets/7776147/e54714ed-1316-4eb8-b933-39d3cf06bd48)